RecoTracker/Configuration/python/customizeMinPtForHitRecoveryInGluedDet.py

```diff
@@ -0,0 +1,10 @@
+def esproducers_by_type(process, *types):
+    return (module for module in process._Process__esproducers.values() if module._TypedParameterizable__type in types)
+def customizeMinPtForHitRecoveryInGluedDet(process,value):
+   for esp in esproducers_by_type(process, "Chi2MeasurementEstimatorESProducer", "Chi2ChargeMeasurementEstimatorESProducer"):
+       esp.MinPtForHitRecoveryInGluedDet = value
```

Comment:
I think the HLT policy for new PRs is to support the old/current menu.
So, missing required parameters should be inserted with a customization.
This is a bit special case, because we are discussing the customization function which is not a standard workflow.